consecutive-count/io.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "input": [1, 2, 2, 3, 4, 4, 4],
```

Comment:
If you want the input to be a list, you need to use double [[ ]]
so [1, 2, 3] will be [[1, 2, 3]]

Reply:
@toranj2006 @SeriousVincyvincy